sys/net/gnrc/network_layer/sixlowpan/frag/sfr/Kconfig

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2020 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_GNRC_SIXLOWPAN_FRAG_SFR
+    bool "Configure GNRC 6LoWPAN Selective Fragment Recovery"
+    depends on USEMODULE_GNRC_SIXLOWPAN_FRAG_SFR
+    help
+        Configure GNRC 6LoWPAN Selective Fragement Recovery using Kconfig.
+
+if KCONFIG_USEMODULE_GNRC_SIXLOWPAN_FRAG_SFR
+
+config GNRC_SIXLOWPAN_SFR_MIN_FRAG_SIZE
+    int "Default minimum value for fragment size (MinFragmentSize)"
+    default 96
+
+config GNRC_SIXLOWPAN_SFR_MAX_FRAG_SIZE
+    int "Default maximum value for fragment size (MaxFragmentSize)"
+    default 112
+    help
+        It must be lower than the minimum MTU along the path. A large value
+        augments the chances of buffer bloat and transmission loss. The value
+        must be less than 512 if the unit is defined for the PHY layer is the
+        octet.
+
+config GNRC_SIXLOWPAN_SFR_OPT_FRAG_SIZE
+    int "Default value for fragment size that the sender should use to start with (OptFragmentSize)"
+    default 112
+    help
+        Must be inclusively between @ref GNRC_SIXLOWPAN_SFR_MIN_FRAG_SIZE and
+        @ref GNRC_SIXLOWPAN_SFR_MAX_FRAG_SIZE
+
+config GNRC_SIXLOWPAN_SFR_USE_ECN
+    bool "Indicates whether the sender should react to ECN (UseECN)"
+    default false
+    help
+        When the sender reacts to ECN its window size will vary between @ref
+        CONFIG_GNRC_SIXLOWPAN_SFR_MIN_WIN_SIZE and @ref
+        CONFIG_GNRC_SIXLOWPAN_SFR_MAX_WIN_SIZE.
+
+if GNRC_SIXLOWPAN_SFR_USE_ECN
+comment "Warning: Reaction of sender to ECN is not implemented yet"
+endif
+
+config GNRC_SIXLOWPAN_SFR_MIN_WIN_SIZE
+    int "Default minimum value of window size that the sender can use (MinWindowSize)"
+    default 1
+    range 1 32
+
+config GNRC_SIXLOWPAN_SFR_MAX_WIN_SIZE
+    int "Default maximum value of window size that the sender can use (MaxWindowSize)"
+    default 16
+    range 1 32
+
+config GNRC_SIXLOWPAN_SFR_OPT_WIN_SIZE
+    int "Default value of window size that the sender should start with (OptWindowSize)"
+    default 16
+    range 1 32
+    help
+        Must be inclusively between @ref GNRC_SIXLOWPAN_SFR_MIN_WIN_SIZE and
+        @ref GNRC_SIXLOWPAN_SFR_MAX_WIN_SIZE
+
+config GNRC_SIXLOWPAN_SFR_INTER_FRAME_GAP_US
+    int "Default minimum amount of time between transmissions in microseconds (InterFrameGap)"
+    default 100
```

Comment:
I wonder if this could be further optimized: I think we always get a TX done event, so we could measure the time it took to TX the frame and assume the next hop will take as much time for TXing it.

Reply:
Sure, it is [mentioned in the RFC](https://www.rfc-editor.org/rfc/rfc8931.html#name-considerations-on-congestio), that the IFG is one of the factors that can be used for congestion control. Adapting this based on radio events, can be part of a congestion control strategy. It should however noted, that the IFG also plays into collision avoidance, so to take hidden terminal problems into account, it might not be enough to take the neighbor's TX time into account, but also its neighbors TX time too.

Reply:
At any rate: this is more a thing for future adaptations, than for this PR.